src/shared_utilities/models.jl

```diff
@@ -41,7 +41,7 @@ abstract type AbstractImExModel{FT} <: AbstractModel{FT} end
     AbstractExpModel{FT} <: AbstractModel{FT}
 
 An abstract type for models which must be treated explicitly.
-This inherits all the default function definitions from AbstractModel, as well
+This inherits all the default function definitisons from AbstractModel, as well
```

Comment:
typo

Reply:
will fix and then squash and bors r+!

thanks for the quick review :)